### PR TITLE
DIGCOLL-5750 New basic "Jump To..." banner

### PR DESCRIPTION
**Pull request by davidseguin**
This will be the first component in an updated banner. Details here: https://jira.metmuseum.org/browse/DIGCOLL-5750

### CODE REVIEW

**Comment by stephenhmarsh** [comment]
Worth mentioning: Our focus states on links are just a thin underline, not really accessible at all, but that's not the fault of this branch. 

Screenshot 2022-12-09 at 1 38 16 PM

File: src/components/jump-to-banner/jump-to-banner.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+			let jumpToLinks = jumpToBanner.querySelectorAll(".js-jump-to__link");
+	
+			jumpToLinks.forEach((jumpToLink) => {
+				jumpToLink.addEventListener("click", function (e) {
+					e.preventDefault();
+					handleLinkClick(jumpToLink);
+				});
+			});
+
+			const handleLinkClick = (jumpToLink) => {
```
Comment (stephenhmarsh) [inline src/components/jump-to-banner/jump-to-banner.js:15]: hm, i don't see any _need_ for this handler to be defined here?  if you could move it out of this function to the top level and just reference it, that would cut down on this function being 27 lines long.

File: src/components/jump-to-banner/jump-to-banner.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const jumpToBanner = () => {
+	if (document.querySelector(".js-jump-to")) {
+		const jumpToBanners = document.querySelectorAll(".js-jump-to");
+
+		jumpToBanners.forEach((jumpToBanner)=> {
+			let jumpToLinks = jumpToBanner.querySelectorAll(".js-jump-to__link");
```
Comment (stephenhmarsh) [inline src/components/jump-to-banner/jump-to-banner.js:6]: nitpick: this value doesn't change, so i think `const` is more appropriate

File: src/components/jump-to-banner/jump-to-banner.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+const jumpToBanner = () => {
+	if (document.querySelector(".js-jump-to")) {
+		const jumpToBanners = document.querySelectorAll(".js-jump-to");
+
+		jumpToBanners.forEach((jumpToBanner)=> {
```
Comment (stephenhmarsh) [inline src/components/jump-to-banner/jump-to-banner.js:5]: do we need to iterate through the _banners_? if you think about it, all the logic is nicely scoped to the individual link.  you could just iterate through those without this forEach.

**Review by stephenhmarsh — CHANGES_REQUESTED** [review]
nice but just have a few comments, thanks (:

File: src/components/jump-to-banner/jump-to-banner.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+					handleLinkClick(jumpToLink);
+				});
+			});
+
+			const handleLinkClick = (jumpToLink) => {
+				console.log("link clicked", jumpToLink);
+				let urlFromLink = new URL(jumpToLink.href);
+				let idFromURL = urlFromLink.hash.substring(1);
+				let sectionToScrollTo = document.getElementById(idFromURL);
+				window.scrollTo({
```
Comment (stephenhmarsh) [inline src/components/jump-to-banner/jump-to-banner.js:20]: you might want to consider https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView which I think would respect CSS scroll-margin

**Review by stephenhmarsh — APPROVED** [review]

**Review by JosephCurley — APPROVED** [review]

**Pull request merged**